onnxscript/tests/function_libs/torch_lib/ops_test.py

```diff
@@ -303,7 +321,10 @@ def setUp(self) -> None:
     def test_output_match_opinfo_(
         self, device: str, dtype: torch.dtype, op: opinfo_core.OpInfo
     ):
-        """Base test method for testing each op with the eager executor, used by instantiate_device_type_tests."""
+        if not _dtype_is_supported_by_op(op.name, dtype):
+            return
```

Comment:
Prefer skipping or else we mistake it as a pass?

Reply:
changed to use pytest.skip()